src/Tests/Microsoft.NET.Publish.Tests/NetCorePublishItemsOutputGroupTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class NetCorePublishItemsOutputGroupTests : SdkTest
+    {
+        public NetCorePublishItemsOutputGroupTests(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        private static List<string> FrameworkAssemblies = new List<string>()
+        {
+            "api-ms-win-core-console-l1-1-0.dll",
+            "System.Runtime.dll",
+            "WindowsBase.dll",
+        };
+
+        [CoreMSBuildOnlyFact]
+        public void GroupPopulatedWithRid()
+        {
+            var testProject = this.SetupProject();
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var restoreCommand = new RestoreCommand(Log, testAsset.Path, testProject.Name);
+            restoreCommand
+                .Execute()
+                .Should()
+                .Pass();
+
+            var buildCommand = new BuildCommand(Log, testAsset.Path, testProject.Name);
+            buildCommand
+                .Execute("/p:RuntimeIdentifier=win-x86", "/t:NetCorePublishItemsOutputGroup")
+                .Should()
+                .Pass();
+
+            var testOutputDir = Path.Combine(testAsset.Path, testProject.Name, "TestOutput");
+            Log.WriteLine("Contents of NetCorePublishItemsOutputGroup dumped to '{0}'.", testOutputDir);
+
+            // Check for the existence of a few specific files that should be in the directory where the 
+            // contents of NetCorePublishItemsOutputGroup were dumped to make sure it's getting populated.
+            Assert.True(File.Exists(Path.Combine(testOutputDir, $"{testProject.Name}.exe")), $"Assembly {testProject.Name}.exe is present in the output group.");

```

Comment:
This is only true on Windows.  You'll need to use the `Constants.ExeSuffix` (e.g. `$"{testProject.Name}{Constants.ExeSuffix}"`) which is `.exe` on Windows and empty string on other platforms.

This appears to be why the tests are failing.

Reply:
Erm, sorry.  This comment was meant for the assertion down below that doesn't have the RID.